README.md

```diff
@@ -204,6 +210,14 @@ import ExamplesList from './ExamplesList';
 import ExampleView from './ExampleView';
 ```
 
+#### `importOrderMergeDuplicateImports`
```

Comment:
Can you explain your thoughts on why this should be an option?  @blutorange and I both believe that this project already has too many options, and we'd like to be more opinionated and rely less on having to set options to get reasonable results.  I think we shouldn't be afraid to make breaking changes and releasing major versions, if needed.